website/app/routes/show.js

```diff
@@ -128,7 +135,7 @@ export default class ShowRoute extends Route {
           });
         }
 
-        return {
+        const model = {
```

Comment:
you can remove one computation, if you want
```suggestion
        this.modelCache[path] = {
```